<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文中宋&quot;;
color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad,  x1:0, x2:0, y1:0, y2:1,  
                                      stop: 0 rgb(209, 255, 202),
                                      stop: 0.495 rgb(170, 255, 127),
                                      stop: 0.505 rgb(170, 255, 127), 
                                      stop: 1 rgb(209, 255, 202));
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="startDateLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>129</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;学期开始日期&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="startDateEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>154</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalWeeksLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>108</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;学期总周数&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="totalWeeksEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>56</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="darkModeLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>86</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;界面颜色&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="darkModeButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>109</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>暗色模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="helpLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>43</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;帮助&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>211</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>点击查看帮助文档</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
